go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/redhatinsights/yggdrasil
 
-go 1.23.0
+go 1.24.0
```

Comment:
Was this change really necessary? If I understand this correctly, then this line says that `yggdrasil` source code requires (directly or indirectly) go 1.24 to be able to compile it. Does really `yggdrasil` or required package require go 1.24? Isn't change of `toolchain` enough?